browser/brave_wallet/BUILD.gn

```diff
@@ -83,6 +83,18 @@ source_set("brave_wallet_delegate") {
       "//components/permissions",
       "//mojo/public/cpp/bindings",
     ]
+    if (enable_extensions) {
+      sources += [
+        "brave_wallet_importer_delegate_impl.cc",
+        "brave_wallet_importer_delegate_impl.h",
+      ]
+      deps += [
+        "//brave/browser/ethereum_remote_client",
+        "//brave/browser/ethereum_remote_client:browser",
```

Comment:
We should check `ethereum_remote_client_enabled` here too for these 2 deps from `browser/ethereum_remote_client/buildflags/buildflags.gni`

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558